crates/anvil/src/eth/backend/db.rs

```diff
@@ -569,8 +569,8 @@ mod test {
         let block = r#"{
             "header": {
                 "parentHash": "0xceb0fe420d6f14a8eeec4319515b89acbb0bb4861cad9983d529ab4b1e4af929",
-                "ommersHash": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
-                "beneficiary": "0x0000000000000000000000000000000000000000",
+                "sha3Uncles": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
+                "miner": "0x0000000000000000000000000000000000000000",
```

Comment:
This looks similar to https://github.com/foundry-rs/foundry/issues/9324 in terms of a breaking change

Is there a way we can handle this elegantly?

Reply:
Can add aliases in alloy

Reply:
I think that would be beneficial, it makes sure that users are able to load older Anvil states

Reply:
should we then add the gas limit too?

Reply:
yes we should